src/Controls/src/Core/RadioButton/RadioButton.iOS.cs

```diff
@@ -8,10 +8,44 @@ public static void MapContent(RadioButtonHandler handler, RadioButton radioButto
 
 		public static void MapContent(IRadioButtonHandler handler, RadioButton radioButton)
 		{
+			// We want to modify the text in the content property only if
+			// the content is a string. In that case we want to apply
+			// the text values from the radio button
+			if (radioButton.Content is string
```

Comment:
Can we resolve this by adding all these properties to templated bindings?

https://github.com/dotnet/maui/blob/b2870c7be752ccf8fa4085a45d28ef2d3200a8f1/src/Controls/src/Core/RadioButton/RadioButton.cs#L462C4-L464